lwt.opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "1.2"
+name: "lwt"
+version: "dev"
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Mauricio Fernandez <mfp@acm.org>"
+  "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+]
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt/manual/"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+dev-repo: "https://github.com/ocsigen/lwt.git"
+build: [ 
+  [ "ocaml" "src/util/configure.ml" "-use-libev" "%{conf-libev:installed}%" ]
+  [ "jbuilder" "build" "-p" name "-j" jobs ] 
+]
+build-test: [ [ "jbuilder" "runtest" "-p" name ] ]
+depends: [
+  "ocamlfind" { build & >= "1.5.0" }
+  "jbuilder" { build & >= "1.0+beta9" }
+  "ocaml-migrate-parsetree" 
```

Comment:
Do we need to depend on omp explicitly? I think `ppx_tools_versioned` pulls that in. However, I'm not yet familiar with the best practices for these packages.